test/Makefile

```diff
@@ -12,7 +12,7 @@ all:
 	$(MAKE) zdtm-freezer
 .PHONY: all
 
-TESTS = unix-callback mem-snap rpc libcriu mounts/ext security pipes crit socketpairs overlayfs mnt-ext-dev shell-job
+TESTS = unix-callback mem-snap rpc libcriu mounts/ext security pipes crit socketpairs overlayfs mnt-ext-dev shell-job skip-file-rwx-check
```

Comment:
It would be good to update [`run-ci-tests.sh`](https://github.com/checkpoint-restore/criu/blob/2b3763fe6a39033d4e3d3d80e28911d1737ac23b/scripts/ci/run-ci-tests.sh#L304) to run the test in CI as well.

Reply:
Missed that, thanks. Done.